plugin-migration/Jenkinsfile

```
@@ -16,7 +16,7 @@ pipeline {
     }
   }
   stages {
-    stage('Generate GitHub Permissions Report') {
+    stage('Generate Plugin-Migration Report') {
```

```suggestion
    stage('Generate Plugin Documentation Migration Report') {
```
Cf https://github.com/jenkins-infra/infra-reports/blob/6f6c3974d724ed1e4753ecea314c4630787e53b9/README.md#L80